tests/stdlib/test_math.py

```diff
@@ -0,0 +1,48 @@
+from unittest import expectedFailure
+
+from ..utils import TranspileTestCase
+
+
+class MathModuleTests(TranspileTestCase):
+    def test_fabs_float_negative(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.fabs(-1.4))
+            """)
+
+    def test_fabs_float_zero(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.fabs(0.0))
+            """)
+
+    def test_fabs_float_positive(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.fabs(1.4))
+            """)
+    
+    def test_fabs_integer_negative(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.fabs(-1))
+            """)
+
+    def test_fabs_integer_zero(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.fabs(0))
+            """)
+
+    def test_fabs_integer_positive(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.fabs(1))
+            """)
+
+    @expectedFailure
```

Comment:
With `@expectedFailure` we don't check that both versions return the same exception or even that both fail. This can be fixed by instead using:
```
try:
  print(math.fabs('string'))
except Exception as e:
  print(type(e), ':', e)
```
This ensures that both versions output the error in the same format.

Reply:
That is a good idea. The error message is now checked (see commit da38165).